### DISCUSSION TITLE
When I get https://localhost.sharedwithexpose.com, does that mean I'm the only one using it?

### DISCUSSION THREAD

**Discussion by johnRivs** [exposedev/expose#375] [Q&A] [answered]
If me and somebody else both get `https://localhost.sharedwithexpose.com` when `expose share http://localhost:8000`, how does a visitor get to the intended website? Can 2 people get that URL in the first place?

**Comment by mpociot** [comment, marked as answer]
No, the server checks if the subdomain is already taken

**Reply by johnRivs** [reply]
Haha, figured. Guess I'm getting lucky lately.
